dockstore-webservice/src/main/java/io/dockstore/webservice/resources/HostedWorkflowResource.java

```diff
@@ -177,10 +183,15 @@ public Workflow editHosted(User user, Long entryId, Set<SourceFile> sourceFiles)
     @Path("/hostedEntry/{entryId}")
     @Timed
     @UnitOfWork
-    @ApiOperation(nickname = "addZip", value = "Creates a new revision of a hosted workflow from a zip",
-            authorizations = {@Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Workflow.class)
-    public Workflow addZip(@ApiParam(hidden = true) @Auth User user, @ApiParam(value = "hosted entry ID")
-        @PathParam("entryId") Long entryId,  @FormDataParam("file") InputStream payload) {
+    @ApiOperation(nickname = "addZip", value = "Creates a new revision of a hosted workflow from a zip", authorizations = {
+        @Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Workflow.class)
+    @Deprecated(since = "1.9.0")
+    @Operation(operationId = "addZip", summary = "Creates a new revision of a hosted workflow from a zip", security = @SecurityRequirement(name = "bearer"), deprecated = true)
+    @ApiResponse(responseCode = "200", description = "successful operation", content = @Content(mediaType = "application/json", schema = @Schema(implementation = Workflow.class)))
```

Comment:
MediaType.APPLICATION_JSON

Reply:
Also, see if you can find a responseCode String somewhere. I was not able to

Reply:
I usually use HttpStatus.SC_OK

Reply:
Isn't that an int?

Reply:
Ah, rats

Reply:
I wasn't able to find a 200 response code string anywhere too